menu/reader.js

```diff
@@ -0,0 +1,36 @@
+// Import Dependencies
+const fs = require("fs");
+const YAML = require("js-yaml");
+
+class Menu {
+    constructor(text) {
+        this.delivery = text.Delivery;
+        this.instructions = text?.Instructions; // If instructions provided
+        this.dishes = text.Menu;
+        this.modifiers = text.Modifiers
+    }
+
+}
+
+// Convert .yml file to object
+const ymlFileToDict = (filename) => {
+    try {
+        const raw = fs.readFileSync(filename);
```

Comment:
could you look into what's the return type of readFileSync (is it a utf-8 string?)